UPGRADE-4.0.md

```diff
@@ -302,14 +302,12 @@ Form
    Before:
 
    ```php
-   'choice_value' => new PropertyPath('range'),
```

Comment:
the introduction line above still mentions `in favor of passing PropertyPath instances`, without an example that's still confusing right... :thinking:

Reply:
the introduction line has been changed.